includes/class-admin.php

```diff
@@ -55,6 +55,17 @@ public function settings( $fields ) {
 			'std' => 'user_nicename'
 		);
 
+		$fields['checkout_referrals_affiliates_sorting_order'] = array(
+			'name' => __( 'Affiliates Sorting Order', 'affiliatewp-checkout-referrals' ),
+			'desc' => __( 'How the affiliates will be sorted on the checkout dropdown', 'affiliatewp-checkout-referrals' ),
+			'type' => 'radio',
+			'options' => array(
+				'affiliate_id' 	=> 'Affiliate ID',
+				'alphabetical' 	=> 'Alphabetical'
```

Comment:
```suggestion
				'affiliate_id' 	=> __( 'Affiliate ID', 'affiliatewp-checkout-referrals' ),
				'alphabetical' 	=> __( 'Alphabetical', 'affiliatewp-checkout-referrals' ),
```
The option labels need to be translatable.